### PR TITLE
Feat/search

### PR DESCRIPTION
**Pull request by Scharpp**
Selector-bar has a temporary fix in it, the way that the nav-section is being centered is making it a headache right now

### CODE REVIEW

File: src/routes/Search.svelte [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  ];
+
+  let lessonsSearchedFor: { title: string; link: string }[] = [];
+
+  const updateSearch = () => {
+    if (currentInput.trim() === "") {
+      lessonsSearchedFor = [];
+    } else {
+      lessonsSearchedFor = lessons.filter((lesson) =>
+        lesson.title.toLowerCase().includes(currentInput.toLowerCase())
```
Comment (Wrench56) [inline src/routes/Search.svelte]: I will have to create an API for this. Maybe connect tags to topics/units/lessons and search for them.

File: vite.config.ts [hunk 2]
```diff
@@ -2,5 +2,10 @@ import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vite';
 
 export default defineConfig({
-	plugins: [sveltekit()]
+	plugins: [sveltekit()],
+	server: {
+		fs: {
+			allow: ['..'],
+		}
+	}
```
Comment (Wrench56) [inline vite.config.ts:10]: I have no idea what this is. Can you explain?

Comment (Scharpp) [reply vite.config.ts:10]: Required for the svg search to display. https://vitejs.dev/config/server-options.html#server-fs-allow

File: src/routes/Search.svelte (hunk 1 above)
Comment (Scharpp) [reply src/routes/Search.svelte]: Updated, see if that was what you wanted

**Review by Wrench56 — APPROVED** [review]
Thank you so much for changing this! Approved.

**Pull request merged**